form.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+    
+        <section class="container">
+            
+            <fieldset>
+                <legend>Personal information</legend>
+
+                <div class="form-field">
+
+                    <label>
+                        Surname:
+                        <input type="text" name="surmame" required> 
+        
+                    </label>
+                   
+                </div>
+        
+                <div class="form-field">
+        
+                    <label>
+                        Name:
+                        <input type="text" name="name" required>
+        
+                    </label>
+                   
+                </div>
+        
+                <div class="form-field">
+        
+                    <label>
+                        How old are You?
+                    <input type="number" name="age" placeholder="12" 
+                           min="0" max="100">
```

Comment:
```suggestion
                    <input
                      type="number"
                      name="age"
                      placeholder="12" 
                      min="0"
                      max="100"
                    >
```
Форматування
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation